index.js

```diff
@@ -2485,17 +2550,14 @@
                def);
   }
 
-  var create =
-  def('create',
-      {},
-      [RecordType({checkTypes: Boolean_, env: Array_(Any)}), AnyFunction],
-      _create);
+  var Options = RecordType(Object_, {checkTypes: Boolean_, env: Array_(Any)});
```

Comment:
Could this be `Array_(Type)` rather than `Array_(Any)`? If there's a good reason for using `Any` rather than `Type`, I've forgotten it. ;)